test/unittest/integration_tests/meson.build

```diff
@@ -18,7 +18,9 @@ fsu_targets = [
 
 if get_option('enable-fp16')
     test_target += mixed_precision_targets
+    if get_option('enable-memory-swap')
```

Comment:
I guess we need to change the name of the option : enable-fsu

Reply:
Yes!! i will update soon swap -> fsu change PR